packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -564,8 +584,26 @@ abstract class ImageProvider<T extends Object> {
   ///
   ///  * [ResizeImage], for modifying the key to account for cache dimensions.
   @protected
+  @Deprecated(
+    'Implement loadBuffer for faster image loading. '
+    'This feature was deprecated after v2.13.0-1.0.pre.',
+  )
```

Comment:
Implement this to throw instead of making implementations throw.

Reply:
Done